code/_onclick/hud/screen_objects.dm

```diff
@@ -11,7 +11,8 @@
 	icon = 'icons/mob/screen_gen.dmi'
 	layer = HUD_LAYER
 	plane = HUD_PLANE
-	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF
+	resistance_flags = INDESTRUCTIBLE
```

Comment:
I went ahead and changed all things that had a resistance flag of `LAVA_PROOF` or `FIRE_PROOF` to be a `burn_state` instead.

Mixing the two results in incorrect flag logic; when I originally had tendrils with `resistance_flags = LAVA_PROOF | FIRE_PROOF`, they were invincible.